x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/new_terms.ts

```diff
@@ -173,8 +177,9 @@ export default ({ getService }: FtrProviderContext) => {
         'kibana.alert.workflow_tags': [],
         'kibana.alert.workflow_assignee_ids': [],
         'kibana.alert.depth': 1,
-        'kibana.alert.reason':
-          'authentication event with source 8.42.77.171 by root on zeek-newyork-sha-aa8df15 created high alert Query with a rule id.',
+        'kibana.alert.reason': expect.stringMatching(
+          /authentication event with source 8\.42\.77\.171 by (root|bob) on zeek-newyork-sha-aa8df15 created high alert Query with a rule id\./
```

Comment:
Is this now matching multiple events due to the non deterministic sort order bug for new terms?

Reply:
yes, that's right.
There are 2 events with the same timestamp https://github.com/elastic/kibana/issues/180236#issuecomment-2042291205

We can go further, and just pass `expect.any(String)`, but I think this implementation should be fine